test-resources/openapi-service-specs/swagger-yaml-service.yml

```diff
@@ -41,7 +41,8 @@ paths:
       responses:
         '200':
           description: Success
-          type: string
+          schema:
+            type: string
```

Comment:
the real fix

Reply:
Good catch, I though this was 2.0 syntax, but I crosschecked the docs and it isn't ;)